src/qml/main.qml

```diff
@@ -310,7 +319,7 @@ Kirigami.ApplicationWindow {
         id: namePicker;
 
         function checkTailName(deviceID) {
-            if (!AppSettings.deviceNames[deviceID]) {
+            if (deviceID && !AppSettings.deviceNames[deviceID]) {
```

Comment:
Hmm... was about to suggest this would be falsy if the ID is zero (mmm, javascript's boolean concept is so lovely), but... that would also not be right, so we can do that, i guess, and it does clear up a bit of an issue.

Reply:
At JavaScript and QML it is absolutely correct to do validations in that way. Also, as I told before we can check Boolean values like this: `if (isValue) {}` instead of `if (isValue === true)` or `if (!pointer)` instead of `if (pointer === null)`. It is easier to read and understand and does not lead to issues :)